tests/thread_tests/remote.go

```diff
@@ -142,6 +142,12 @@ func remote_exec(t *testing.T, command string) string {
 
 	t.Logf("[exec-ssh] %s", command)
 
+	// Remote commands that require sudo might ask for the password. Always pass it in. See https://stackoverflow.com/a/11955358
+	if strings.HasPrefix(command, "sudo ") {
+		command = command[5:] // remove "sudo "
+		command = "echo \"" + remotePassword + "\" | sudo -S " + command
```

Comment:
`echo \"" + remotePassword + "\"`

If the password contains a `"`-character, this will fail.